from app.application.base.exceptions import ApplicationError


class AuthorizationError(ApplicationError):
    pass
